nullplatform/metadata_specification.go

```diff
@@ -0,0 +1,116 @@
+package nullplatform
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+)
+
+const (
+	METADATA_SPECIFICATION_PATH = "/metadata_specification"
```

Comment:
```suggestion
	METADATA_SPECIFICATION_PATH = "/metadata/metadata_specification"
```
De acuerdo a su documentación[^1] el ¿path debería ser `/metadata/metadata_specification`?

[^1]: https://docs.nullplatform.com/api/create-a-entity-metadata-specification